### PR TITLE
Adds code to protect parameters

### PR DESCRIPTION
**Pull request by DrPaulSharp**

### CODE REVIEW

File: tests/testProjectConversion/originalDSPCBilayerProjectClass.mat
Comment (StephenNneji) [inline tests/testProjectConversion/originalDSPCBilayerProjectClass.mat:1]: I think this file was renamed

**Review by StephenNneji — COMMENTED** [review]
Please add test for the protected parameters

File: tests/testProjectConversion/originalDSPCBilayerProjectClass.mat
Comment (DrPaulSharp) [reply tests/testProjectConversion/originalDSPCBilayerProjectClass.mat:1]: In PR #127 right?

Comment (StephenNneji) [reply tests/testProjectConversion/originalDSPCBilayerProjectClass.mat:1]: Sorry, I forgot this was not merged

**Review by StephenNneji — APPROVED** [review]

**Pull request merged**